Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -21,6 +21,8 @@ Shader "Mixed Reality Toolkit/Standard"
         [Toggle(_EMISSION)] _EnableEmission("Enable Emission", Float) = 0.0
         [HDR]_EmissiveColor("Emissive Color", Color) = (0.0, 0.0, 0.0, 1.0)
         [Toggle(_TRIPLANAR_MAPPING)] _EnableTriplanarMapping("Triplanar Mapping", Float) = 0.0
+        [Toggle(_USE_SSAA)] _EnableSSAA("Super Sample Anti Aliasing", Float) = 0.0
+        _MipmapBias("Mipmap Bias", Range(-1.0, 0.0)) = 0.0
```

Comment:
I found having a larger range (-10, 0) provided some "crisper" results. But, I haven't spent time to see what other effects are larger bias could have.

One "end user" suggestion would be to have the range be positive then negate the bias in the shader (since it's weird to think in negative biases). But, this suggestion is totally up to you.

Reply:
Also the bias defaults to zero, so when a user turns on _EnableSSAA so they don't see an immediate change. Might be nice to have a sane "default value" other than zero.

Reply:
Yeah there's some more work I want to do to verify the quality of the implementation, I'll keep you posted :)